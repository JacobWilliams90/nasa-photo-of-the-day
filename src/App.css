.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

img {
  max-width: 1600px;
  max-height: 900px;
}

body {
  margin: 0 auto;
  background: black;
  color: white;
  width: 1080px;
}

.header {
  margin: 0 auto;
  display: flex;
  justify-content: space-around;
  align-items: baseline;
}

.foot{
  font-size: 1.5rem;
}

a {
  text-decoration: none;
  font-size: 1.5rem;
}

a:visited {
  color: blue;
}

a:hover {
  color: white;
}
