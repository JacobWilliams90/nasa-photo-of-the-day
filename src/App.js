import React from "react";
import "./App.css";
import Header from './head'
import Body from './body'
import Footer from './foot'
import { StyledPage } from './StyledPage';


function App() {
  return (
    <StyledPage>
      <Header />
      <Body />
      <Footer />
    </StyledPage>
  );
}

export default App;
