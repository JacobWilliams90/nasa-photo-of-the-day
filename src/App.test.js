// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
// import { exportAllDeclaration } from '@babel/types';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);

// });


import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
