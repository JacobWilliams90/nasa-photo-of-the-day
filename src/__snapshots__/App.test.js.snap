// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<html
  className="sc-bdVaJa etRxVd"
>
  <div
    className="sc-htpNat dGbXDD"
  >
    <h1>
      Nasa Photo of the day
    </h1>
    <button
      className="sc-bwzfXH kbGHtW"
    >
      <a
        href="http://www.nasa.gov"
      >
        Take me back to NASA
      </a>
    </button>
  </div>
  <h3>
    Loading...
  </h3>
  <div
    className="sc-EHOje tnHhq"
  >
    <p>
      Copyright 2019
    </p>
  </div>
</html>
`;
